bin/install-version

```diff
@@ -17,12 +17,15 @@ if [ ! "$#" -eq 1 ]; then
 	exit 1
 fi
 
+INSTALL_LOCATION=$HOME/.minimesos/bin
 VERSION=$1
-echo "VERSION GIVEN IS " $VERSION
-
-curl -sSL https://raw.githubusercontent.com/ContainerSolutions/minimesos/$VERSION/bin/minimesos > /usr/local/bin/minimesos
-chmod +x /usr/local/bin/minimesos
-
-# TODO: pull docker image from Docker Hub
+echo "Installing version " $VERSION
+mkdir -p $INSTALL_LOCATION
+curl -sSL https://raw.githubusercontent.com/ContainerSolutions/minimesos/$VERSION/bin/minimesos > $INSTALL_LOCATION/minimesos
+chmod +x $INSTALL_LOCATION/minimesos
+
+echo "minimesos is installed into ${INSTALL_LOCATION}/minimesos"
+echo "Run the following command to add it to your executables path:"
+echo "export PATH=\$PATH:$INSTALL_LOCATION"
```

Comment:
How is about `echo "export PATH=$INSTALL_LOCATION:\$PATH"`

Otherwise old version of minimesos, which might be in `/usr/bin` will be invoked